Source/WebCore/loader/SubresourceLoader.cpp

```diff
@@ -633,7 +638,7 @@ static void logResourceLoaded(LocalFrame* frame, CachedResource::Type type)
 
 Expected<void, String> SubresourceLoader::checkResponseCrossOriginAccessControl(const ResourceResponse& response)
 {
-    if (!m_resource->isCrossOrigin() || options().mode != FetchOptions::Mode::Cors)
+    if (!protectedCachedResource()->isCrossOrigin() || options().mode != FetchOptions::Mode::Cors)
```

Comment:
I think it is probably OK because it is not hot code but you don't really need to ref to call trivial getters.

Reply:
Fixed.